doc/api/animation_api.rst

```diff
@@ -124,7 +125,36 @@ artist at a global scope and let Python sort things out.  For example ::
    plt.show()
 
 The second method is to use `functools.partial` to 'bind' artists to
```

Comment:
I think `bind` is doing too much work here. What about partial is used here to pass arguments to the update method - which separately is kind of confusing since this update method only takes one argument,

Reply:
So, increase the number of arguments?

Reply:
I think it could make it clearer what partial is doing here